go/libraries/doltcore/table/editor/index_editor.go

```diff
@@ -93,16 +94,16 @@ type indexEditAccumulator struct {
 	rowData types.Map
 
 	nbf     *types.NomsBinFormat
-	ed      types.EditAccumulator
-	opCount uint64
+	opCount int64
 
 	// addedPartialKeys is a map of partial keys to a map of full keys that match the partial key
 	addedPartialKeys map[hash.Hash]map[hash.Hash]types.Tuple
-	addedKeys        map[hash.Hash]struct{}
-	removedKeys      map[hash.Hash]struct{}
+	addedKeys        map[hash.Hash]hashedTuple // These hashes represent the hash of the partial key, with the tuple being the full key
+	removedKeys      map[hash.Hash]hashedTuple // These hashes represent the hash of the partial key, with the tuple being the full key
```

Comment:
I worry about the size of these two maps.

Reply:
It shouldn't be any different than before since I removed `affectedKeys`, and that contained all of the entries anyway.